client/state/data-layer/wpcom/read/teams/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import { READER_TEAMS_REQUEST, READER_TEAMS_RECEIVE } from 'state/action-types';
+import wpcom from 'lib/wp';
+
+export function handleTeamsRequest( store, action, next ) {
+	wpcom.req.get( '/read/teams', { apiVersion: '1.2' } )
+		.then(
+			payload => {
```

Comment:
> sneaky naming it payload

actually yes :smile:

> i like it

I'm not so much a fan. something I think is really critical to the data layer is that provide _in the Calypso code_ a place that describes that the data should look like going into and coming out of the app. it may well be the case that the API _right now_ sends the data in the form that Calypso wrapped itself around but we don't have a guarantee.

since we had a discussion in Slack and I'm preparing a p2 post on this I won't rehash here, but thanks especially @samouri for reasoning with me about this, challenging what I'm saying, and being willing to explore it